runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -53,6 +51,12 @@
     private long numEntries = 0;
     private long numBytes = 0;
 
+    // Registry and Timer used for measuring append checkpoint
+    private static MetricRegistry metricRegistry = CorfuRuntime.getDefaultMetrics();
```

Comment:
Shouldn't this be `runtime.getMetrics()` similar to AddressSpaceView?

Reply:
@zalokhan It just need to be a static registry like AbstractTransactionalContext or ServerContext.